tests/workflows/basic_tool_calling_node_inline_workflow/tests/test_workflow.py

```diff
@@ -325,3 +325,58 @@ def generate_prompt_events(*args, **kwargs) -> Iterator[ExecutePromptEvent]:
         "expand_meta": None,
         "request_options": mock.ANY,
     }
+
+
+def test_workflow_stream_initiated_events(vellum_adhoc_prompt_client, mock_uuid4_generator):
```

Comment:
This test is perfect. Now take a stab at solving this on the implementation side. We likely need to update the inline subworkflow tool from `run()` to `stream()` so that we could emit events through the WorkflowContext